api/models/vpp/interfaces/interface.proto

```diff
@@ -226,3 +227,16 @@ message BondLink {
     }
     repeated BondedInterface bonded_interfaces = 12;
 }
+
+message GreLink {
```

Comment:
This should be included in the `oneof` field of `Interface` message.

Reply:
yeah, removed it by accident while resolving conflicts :unamused: